content/Box1: experimental design

```diff
@@ -0,0 +1,23 @@
+# Designing experiments for applied machine learning
+
+Machine learning algorithms were and continue to be developed to identify patterns that explain or fit a given dataset.
+As a result, every machine learning algorithm goes through a training phase (where it tries to learn the patterns present in a dataset), and a testing phase (where a dataset unseen by the model is used to test whether the patterns learned by the model resemble real data).
+So every machine learning algorithm expects the following fundamental parts as input: 1. a training dataset, 2. a testing dataset.
+The training dataset typically consists of a matrix of a large number of samples and a large number of features, where the samples follow a specific pattern of the features which the algorithm tries to learn.
+The testing dataset typically is smaller than the training set, but is expected to contain similar patterns of features that are present in the training set.
+The testing set is generally never exposed to the model during its training phase (_hold-out test dataset_).
+Once a model has completed learning the patterns from the training set, only then it is allowed to use the testing set as input.
+The error of a model on the test set is key to understanding how well the model has learned patterns in a given dataset and how generalizable it may be.
```

Comment:
To keep it more general:

```suggestion
The performance of a model on the test set is key to understanding how well the model has learned patterns in a given dataset and how generalizable it may be.
```